nservicebus/upgrades/3to4.md

```diff
@@ -146,6 +146,7 @@ NServiceBus PowerShell cmdlets have moved to NServiceBus.PowerShell.dll. See [Ma
 
 `configure.JsonSerializer()` has been obsoleted, please use `Configure.Serialization.Json();` instead. More info on supported serializers [here](/nservicebus/serialization/).
 
+The `XmlSerializer` will now automatically escape outgoing messages containing invalid characters. Version 3 endpoints will not be able to process those messages correctly.
```

Comment:
for starters time based statements like "will now" should never be used in doco. I read it and think "now? you mean when that text was added"?

And another question. in version 3 if there are "invalid characters" will they be correctly encoded and decoded withing Version 3 endpoints?